xml/installation.xml

```diff
@@ -159,44 +159,16 @@
     cluster. For more information, see <xref linkend="sec-remote-munge"/>.
    </para>
    <para>
-    The system roles are only available for new installations of &productname;.
+    The system roles are only available for new installations of the &hpca; Module for &sles;.
    </para>
   </sect1>
   <sect1 xml:id="sec2-upgrade">
-   <title>Upgrading to &productname; &productnumber;</title>
+   <title>Upgrading to &sles; &hpca; Module &productnumber;</title>
    <para>
-    <!--
-        #if <= 15SP2
-    You can upgrade to &productname; &productnumber; from &slsa; 12 SP4 or SP5
-    as well as &slehpca; &prev-version; or &slehpca; 15. When upgrading from
-    &slsa; 12 SP4/SP5, the upgrade will only be performed if the HPC
-    module has been registered prior to upgrading.
-
-   </para>
-   <para>
-    To upgrade from &slsa; 12 to &slehpca; 15, make sure to deregister the
-    HPC module prior to upgrading. To do so, open a root shell and
-    execute:
-   </para>
-   <screen>SUSEConnect -d -p sle-module-hpc/12/<replaceable>ARCH</replaceable></screen>
-   <para>
-    Replace <replaceable>ARCH</replaceable> with the architecture used
-    (<literal>x86_64</literal>, <literal>aarch64</literal>).
-   </para>
-   <para>
-    When migrating to &productname; &productnumber;, all modules not supported by
-    the migration target need to be deregistered. This can be done by
-    executing:
+    In version &productnumber;, &slehpc; (&slehpca;) was removed as a separate product, and is now available as the &hpca; Module for &sles; (&slsa;).
```

Comment:
Whoops, you're right, it should just be a number. Having said that, I think in SP7 I'll probably just remove this sentence anyway, as by then it will just be a simple "You can upgrade from SLES 15 SP6 plus the HPC module to SLES 15 SP7 plus the HPC module."